tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -103,134 +83,96 @@ const updateSettings = async (targetsConfig, user, contactSummary) => {
-      name: 'Smaller Hospital'
-    };
-
-    const user = {
-      username: 'supervisor',
-      password: password,
-      place: parentPlace._id,
-      contact: {
-        _id: 'fixture:user:supervisor',
-        name: 'Supervisor'
-      },
-      roles: ['national_admin'],
-      known: true,
-      language: 'en',
-    };
-
-    const names = [ 'Clarissa', 'Prometheus', 'Alabama', 'Jasmine', 'Danielle' ];
-
+    const parentPlace = placeFactory.place().build({ type: 'district_hospital' });
+    const otherParentPlace = placeFactory.place().build({ type: 'district_hospital' });
+    const user = userFactory.build({ place: parentPlace._id, roles: ['program_officer'] });
+    const names = ['Clarissa', 'Prometheus', 'Alabama', 'Jasmine', 'Danielle'];
     const genPlace = (parent, idx = false) => {
-      const place = {
-        _id: uuid(),
-        type: 'health_center',
-        name: randomString(8),
-        parent: { _id: parent._id },
-        reported_date: moment().valueOf(),
-      };
-
-      const contact = {
-        _id: uuid(),
-        type: 'person',
-        name:  idx === false ? randomString(8) : names[idx],
-        parent: { _id: place._id, parent: place.parent },
-        reported_date: moment().valueOf(),
-      };
-
+      const place = placeFactory.place().build({ type: 'health_center', parent: { _id: parent._id } });
+      const contact = personFactory.build({
+        name: idx === false ? randomString(8) : names[idx],
+        parent: { _id: place._id, parent: place.parent }
+      });
       place.contact = { _id: contact._id, parent: contact.parent };
-
       return [place, contact];
     };
-
     const docs = _.flattenDeep([
       Array.from({ length: 5 }).map((e, i) => genPlace(parentPlace, i)),
       Array.from({ length: 5 }).map(() => genPlace(otherParentPlace)),
     ]);
-
     const genTitle = (title) => ({ en: title });
-
     const docTags = [
       // current targets
       moment().format('YYYY-MM'),
       // next month targets, in case the reporting period switches mid-test
       moment().date(1).add(1, 'month').format('YYYY-MM'),
     ];
 
-    beforeAll(async () => {
+    before(async () => {
       await utils.saveDocs([parentPlace, otherParentPlace]);
       await utils.saveDocs(docs);
-      await utils.createUsers([ user ]);
-      await utils.resetBrowser();
-      await commonElements.goToLoginPageNative();
-      await loginPage.loginNative(user.username, user.password);
-      await commonElements.calmNative();
-    });
-
-    afterAll(async () => {
-      await commonElements.goToLoginPageNative();
-      await loginPage.loginNative(constants.USERNAME, constants.PASSWORD);
-      await commonElements.calmNative();
+      await utils.createUsers([user]);
+      await browser.url('/medic/login');
+      await loginPage.login({ username: user.username, password: user.password });
+      await commonElements.waitForPageLoaded();
     });
 
     const DOCS_TO_KEEP = [
```

Comment:
do we need `[/^form:/]`?

Reply:
@latin-panda, do you mean to keep [/^form:/]? sorry I didn't understand.

Reply:
Yes, this code is deleting all forms definitions from the DB, and I saw in other places that we [usually keep the forms](https://github.com/medic/cht-core/blob/722d4802da6c5c2f9a7f0f2a840446bd39e94d6d/tests/e2e/default/contacts/add-custom-hierarchy.wdio-spec.js#L69), so I was wondering if this is intentional or something we forgot and need to do.

Reply:
Ah okay, yes, this is something we forgot. I will add [/^form:/] to the docs to keep. Thanks!